src/components/Inputs/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,143 @@
+        (_event?: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) =>
+            triggerChange(_event),
+        waitInterval
+    );
+
+    const triggerChange = (
+        _event?: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+    ) => {
+        onChange;
+    };
+
+    return (
+        <div className={textAreaWrapperClassNames}>
+            <div className={styles.fieldLabel}>
+                {label && (
+                    <label className={styles.textStyle} htmlFor={name}>
+                        {label}
+                    </label>
+                )}
+                {labelIconButtonProps && labelIconButtonProps.show && (
+                    <span className={styles.fieldLabelIconButton}>
+                        <Tooltip
+                            content={labelIconButtonProps.toolTipContent}
+                            placement={
+                                labelIconButtonProps.toolTipPlacement
+                                    ? labelIconButtonProps.toolTipPlacement
+                                    : 'top'
+                            }
+                            positionStrategy={
+                                labelIconButtonProps.toolTipPositionStrategy
+                            }
+                            theme={labelIconButtonProps.toolTipTheme}
+                        >
+                            <DefaultButton
+                                allowDisabledFocus={
+                                    labelIconButtonProps.allowDisabledFocus
+                                }
+                                ariaLabel={labelIconButtonProps.ariaLabel}
+                                className={styles.labelIconButton}
+                                disabled={labelIconButtonProps.disabled}
+                                icon={
+                                    labelIconButtonProps.icon
+                                        ? labelIconButtonProps.icon
+                                        : IconName.mdiInformation
+                                }
+                                iconColor={labelIconButtonProps.iconColor}
+                                onClick={
+                                    !labelIconButtonProps.allowDisabledFocus
+                                        ? labelIconButtonProps.onClick
+                                        : null
+                                }
+                                size={ButtonSize.Small}
+                            />
+                        </Tooltip>
+                    </span>
+                )}
+            </div>
+            <textarea
+                aria-disabled={allowDisabledFocus}
+                aria-label={ariaLabel}
+                autoFocus={autoFocus}
+                className={textAreaClassNames}
+                cols={textAreaCols}
+                disabled={disabled}
+                id={id ? id : textAreaId}
```

Comment:
```suggestion
                id={textAreaId}
```
If we make the change according to the above comment